### PR TITLE
Branded IPCs

### PR DESCRIPTION
**Pull request by Lohikar**
This PR adds 2 subtypes of industrial and two subtypes of baseline, one for each cybernetics manufacturer. Sprites by BRAINOS.

other changes:
- IPCs can now be flashed or flashbanged.
- Admin spawn paths have been added for subspecies that did not have them.
- IPCs are no longer slowed by being cold.

### Bishop Cybernetics (BC) ###
species_preview_bishop cybernetics industrial frame

### Hephaestus Industries (HI) ###
species_preview_hephaestus industries industrial frame

### Xion Manufacturing Group (XMG) ###
species_preview_xion manufacturing group industrial frame

### Zeng-Hu Pharmaceuticals (ZH) ###
species_preview_zeng-hu pharmaceuticals industrial frame

Todo:
- [x] Actually add species blurbs for new species
- [x] Differentiate industrial subtypes
- [x] Get markings sprites from BRAINOS.
- [x] Change BC and ZH frames to be baseline subtypes.
- [x] Fix ZH eyes.

Depends on #3897.

### CODE REVIEW

**Comment by skull132** [comment]
This is scarily small amount of changes when you consider the contents 🤔

**Comment by Lohikar** [comment]
A lot of the framework this PR required was added in a previous PR as a general synth refactor.

**Comment by skull132** [comment]
Still DNM, @Lohikar ?

**Comment by Lohikar** [comment]
DNM indefinitely, still waiting on stuff from both Cake and BRAINOS.

**Comment by LordFowl** [comment]
This will further clutter the species selection screen, immensely so because it's 4 additional options. We really need to refactor it so it handles subspecies more tidily. Not a blocking issue (Because there is no alternative currently), just a comment on the state of affairs.

**Comment by CakeIsOssim** [comment]
Everything's good to go in the lore department. Thank you, Lohikar! 👍

File: code/game/antagonist/station/changeling.dm [hunk 1]
```diff
@@ -5,8 +5,18 @@
 	bantype = "changeling"
 	feedback_tag = "changeling_objective"
 	restricted_jobs = list("AI", "Cyborg", "Head of Security", "Captain", "Internal Affairs Agent")
-	protected_jobs = list("Security Officer", "Security Cadet", "Warden", "Detective", "Forensic Technician")
-	restricted_species = list("Baseline Frame","Shell Frame","Industrial Frame")
+
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Forensic Technician")
```
Comment (Lohikar) [inline code/game/antagonist/station/changeling.dm]: Cadet accidentally removed here.

File: code/game/antagonist/station/vampire.dm [hunk 2]
```diff
@@ -7,8 +7,19 @@ var/datum/antagonist/vampire/vamp = null
 	bantype = "vampires"
 	feedback_tag = "vampire_objective"
 	restricted_jobs = list("AI", "Cyborg", "Chaplain", "Head of Security", "Captain", "Internal Affairs Agent")
-	protected_jobs = list("Security Officer", "Security Cadet", "Warden", "Detective", "Forensic Technician")
-	restricted_species = list("Baseline Frame","Shell Frame","Industrial Frame","Diona")
+
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Forensic Technician")
```
Comment (Lohikar) [inline code/game/antagonist/station/vampire.dm]: And here.

File: code/modules/clothing/clothing.dm [hunk 3]
```diff
@@ -106,7 +113,7 @@
 	//Set species_restricted list
 	switch(target_species)
 		if("Skrell")
-			species_restricted = list("Human", "Skrell", "Baseline Frame", "Shell Frame", "Industrial Frame") // skrell helmets like to share
+			species_restricted = list("Human", "Skrell", "Baseline Frame", "Shell Frame", "Hephaestus G1 Industrial Frame") // skrell helmets like to share
```
Comment (Lohikar) [inline code/modules/clothing/clothing.dm]: `Baseline Frame` -> `Machine`

File: code/modules/mob/mob_helpers.dm [hunk 4]
```diff
@@ -72,9 +72,9 @@
 
 /proc/isipc(A)
 	. = 0
-	if(istype(A, /mob/living/carbon/human))
+	if(ishuman(A))
 		var/mob/living/carbon/human/H = A
-		. = !!global.mechanical_species[H.get_species()]
+		. = H.species.flags & IS_MECHANICAL
```
Comment (skull132) [inline code/modules/mob/mob_helpers.dm]: Runtime risk. Do it as the other one, `species && (species.flags & IS_MECHANICAL)`

**Pull request merged**